crates/services/upgradable-executor/src/executor.rs

```diff
@@ -612,38 +664,34 @@ where
     fn get_module(
         &self,
         version: fuel_core_types::blockchain::header::StateTransitionBytecodeVersion,
-    ) -> ExecutorResult<wasmtime::Module> {
+    ) -> Result<wasmtime::Module, UpgradableError> {
```

Comment:
I think this function should return `ExecutorResult` while `validate_uploaded_wasm` resukt with `UpgradableError`. In this case you don't need to change old code

Reply:
ff983661d